lib/autoprefixer.js

```diff
@@ -28,40 +28,52 @@ function timeCapsule(result, prefixes) {
     );
 }
 
-module.exports = postcss.plugin('autoprefixer', (...reqs) => {
+function getOptions(...args) {
     let options;
-    if (reqs.length === 1 && isPlainObject(reqs[0])) {
-        options = reqs[0];
-        reqs = undefined;
-    } else if (reqs.length === 0 || reqs.length === 1 && !reqs[0]) {
-        reqs = undefined;
-    } else if (reqs.length <= 2 && (reqs[0] instanceof Array || !reqs[0])) {
-        options = reqs[1];
-        reqs = reqs[0];
-    } else if (typeof reqs[reqs.length - 1] === 'object') {
-        options = reqs.pop();
+    let reqs;
+
+    // autoprefixer([...,] options)
```

Comment:
I prefer to not write this type of comments. If you need comment, you should make code clearly.

Reply:
I have rewrote a little.